docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,185 @@
 sidebar_position: 2
 ---
 
-# Quick Start
+# Quick start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+Here is a schematic diagram of this setup:
 
-_Here we provided a small sample on K8s_
+![postgresql-spring-boot](/img/docs/postgresql-spring-boot.png)
 
-## OpenShift
+The service binding operator collects backing service configuration required for
+the connectivity and expose it to the sample application.
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+This topic provides the following information on how to deploy application and connect it to a a backing service:
+
+1. [Prerequisites](#prerequisites)
+2. [Creating a database instance](#creating-a-database-instance)
+3. [Deploying an application](#deploying-an-application)
+4. [Connecting the application to a backing service](#connecting-the-application-to-a-backing-service)
+
+## Prerequisites
+
+The following components must be installed and configured:
+
+- Kubernetes cluster (**Note:** You can use [minikube](https://minikube.sigs.k8s.io/) or
+  [kind](https://kind.sigs.k8s.io/), locally)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Service Binding Operator](installing-service-binding).
+
+## Creating a database instance
+
+The application is going to connect to a PostgreSQL database backend.  The
+PostgreSQL can be setup using the [Crunchy PostgreSQL operator from
+OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance for
+connection.
+
+1. To create a database instance, you need to create custom resource
+(`Pgcluster`) and that will trigger the operator reconciliation.  For
+convenience, run this command to create `Pgcluster` custom resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/pgcluster.yaml
```

Comment:
Checking the file here, we can see that you added the annotations for the service binding. but we are not giving any explanations about that and why we put it... 
There is magic or hacks here - and I think we need to explain it: putting the plain yaml and explaining the thing we are doing into it would actually help the reader to understand what's going on!

Reply:
Done. 1c27e60b4ef80faf802e6913a35b59dcc746f62a

Reply:
can you move the yaml under that documentation repo and create a dedicated "resources" folder for that?

Reply:
Done. 0e153c07f9d36cd93a8bd8121c173de04825a7d0